docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -122,7 +135,7 @@ With patches that are not rolling compatible:
 #### Updating deployed applications for domain home in image
```

Comment:
This title needs to be changed since the section applies to both domain home in image and model in image. The content has been changed already to cover the model-in-image use case.

Reply:
Fixed.